include/elliptics/session.hpp

```diff
@@ -317,7 +317,7 @@ class session
 		 * in this session so they won't collide with same key in any
 		 * other namespace.
 		 */
-		void set_namespace(const std::string &ns);
+		void			set_namespace(const std::string &ns);
```

Comment:
I made it consistent with the other surrounding methods, you would notice that if opened the whole session class.
Its first methods which do not accept many arguments are all tab-separated, second half of methods which accept larger number of arguments uses only one space.